tests/run.sh

```diff
@@ -169,6 +169,9 @@ start_awesome() {
     awesome_pid=$!
     cd - >/dev/null
 
+    # Do not put too much load on this, it will deadlock
+    sleep 0.5
```

Comment:
Me being me, I would add a comment "TODO: Figure out what the problem is and fix it properly". But I guess we can also live with a sleep....